automation/check-merged.environment.yaml

```diff
@@ -0,0 +1,5 @@
+- name: REGISTRY_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: quay
+      key: kubevirt_bot
```

Comment:
Once there is a bot created for CI, would you please add just `docker login` to check-merged.sh? We will add the rest then.

Reply:
Sure. Can we do it in a different PR?

Reply:
Fair enough